


Header header

# The poses
int32[] posesId
geometry_msgs/Pose[] poses

# The links
ORB_SLAM2v2/Link[] links



